podcasts/Constants.swift

```diff
@@ -24,6 +24,7 @@ struct Constants {
         static let dimmingViewTapped = NSNotification.Name(rawValue: "SJDimViewTapped")
         static let downloadProgress = NSNotification.Name(rawValue: "SJDwnProg")
         static let podcastImageReCacheRequired = NSNotification.Name(rawValue: "PCPodcastImageReCacheRequired")
+        static let episodeEmbeddedArtworkLoaded = NSNotification.Name(rawValue: "PCEpisodeEmbeddedArtworkLoaded")
```

Comment:
Wdyt about moving this to the extension in [Utilities/Notifications](https://github.com/Automattic/pocket-casts-ios/blob/4d580ed449b313dc25aba508234af8a52b7037be/podcasts/Notifications.swift)?

Reply:
Done in 231ea04bd81de0fabef4cadfc0a359dafc413c92